docs/madr/decisions/082-inspect-api-include-kri-in-rule.md

```diff
@@ -0,0 +1,84 @@
+# Including Policy Origin in Inbound Rules for Inspect API
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/14010
```

Comment:
In other project we actually link the openAPI PR in the MADR. I think it's very helpful can we start doing this here too?

Reply:
https://github.com/kumahq/kuma/pull/14084